ext/fetch/21_formdata.js

```diff
@@ -396,7 +396,7 @@ class MultipartParser {
    */
   constructor(body, boundary) {
     if (!boundary) {
-      throw new TypeError("multipart/form-data must provide a boundary");
+      throw new TypeError("Cannot construct MultipartParser: multipart/form-data must provide a boundary");
```

Comment:
This is not a public API - so this doesn't matter. But it's ok :)

Reply:
While some of these are not public API, do any of these exceptions propagate high enough that the user would see them?